bin/cassandra.in.sh

```diff
@@ -125,9 +125,6 @@ short=$(echo "${jvmver}" | cut -c1-2)
 JAVA_VERSION=17
 if [ "$short" = "11" ]  ; then
      JAVA_VERSION=11
-elif [ "$JVM_VERSION" \< "17" ] ; then
-    echo "Cassandra 5.0 requires Java 11 or Java 17."
-    exit 1;
```

Comment:
The side effect of this removal is that for any version between Java 11 and 17, JAVA_VERSION will be set to 17, no?
Then the jvm options files for Java 17 will be used for versions pre-17. In my opinion we should use the Java 11 files for those JDK versions and the J17 files for any >= J17 version. Look here - https://github.com/apache/cassandra/blob/trunk/bin/cassandra.in.sh#L152-L158

Reply:
+1 use j11 options

Reply:
yes, if JVM_VERSION=16 then it should be JAVA_VERSION=16 so that the jvm11*.options file will be then used.